lib/Serialization/Deserialization.cpp

```diff
@@ -5439,7 +5488,12 @@ ModuleFile::loadAllConformances(const Decl *D, uint64_t contextData,
   DeclTypeCursor.JumpToBit(bitPosition);
 
   while (numConformances--) {
-    auto conf = readConformance(DeclTypeCursor);
+    auto maybeConf = readConformanceChecked(DeclTypeCursor);
+    if (!maybeConf) {
+      llvm::consumeError(maybeConf.takeError());
```

Comment:
This absolutely can't happen until we can drop BoundGenericTypes that rely on the conformance. :-(